sdk/go/common/workspace/project.go

```diff
@@ -157,19 +173,20 @@ type Project struct {
 	AdditionalKeys map[string]interface{} `yaml:",inline"`
 }
 
-func isPrimitiveValue(value interface{}) (string, bool) {
+func isPrimitiveValue(value interface{}) bool {
 	switch value.(type) {
-	case string:
-		return "string", true
-	case int:
-		return "integer", true
-	case bool:
-		return "boolean", true
+	case string, int, bool:
+		return true
 	default:
-		return "", false
+		return false
 	}
 }
 
+func isArray(value interface{}) bool {
+	_, ok := value.([]interface{})
```

Comment:
Is this safe? What if value is a `[]int`?

Reply:
It is safe when checking against unmarshalled values from JSON/YAML which is where this is applicable